docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -140,20 +140,62 @@ Regardless of the way you submit your work, follow these steps.
 .. _ticket tracker: https://code.djangoproject.com/
 .. _Development dashboard: https://dashboard.djangoproject.com/
 
-Non-trivial contributions
-=========================
+Contributions which require community feedback
+==============================================
 
-A "non-trivial" contribution is one that is more than a small bug fix. It's a
-change that introduces new Django functionality and makes some sort of design
-decision.
+A patch that introduces Django functionality and makes some sort of design
+decision requires a wider community discussion. Especially if the approach
+involves a deprecation or potentially introducing breaking changes.
+Please include evidence that the approach and alternatives have been discussed
+on the `Django Forum`_ or |django-developers| list.
 
-If you provide a non-trivial change, include evidence that alternatives have
-been discussed on the `Django Forum`_ or |django-developers| list.
+Here are a few approaches for gaining feedback from the community.
 
-If you're not sure whether your contribution should be considered non-trivial,
-ask on the ticket for opinions.
+Django Forum or the django-developers mailing list
+--------------------------------------------------
+
+You can make a proposal on the `Django Forum`_ or |django-developers| mailing list.
+
+You should explain the need for the change, go into details of the approach
+and discuss alternatives. Please include a link to such discussions in your patch.
+
+Third party package
+-------------------
+
+Django does not accept experimental features. If you need user feedback
+on a public interface, landing this into Django core is not the way to do it.
+Iterations must follow the
+:ref:`deprecation process<internal-release-deprecation-policy>`
+and can take months or even years for the users to see the benefit of such changes.
+
+Therefore, in such cases, it is better to create a third-party package which
+seeks user feedback. The public API can be iterated on and you can validate
+that there is a need for such a feature.
+
+Once this package becomes stable and there are clear benefits of
+incorporating aspects into Django core, starting a discussion on the
+`Django Forum`_ or |django-developers| mailing list would be the next step.
+
+Django Enhancement Proposal (DEP)
```

Comment:
@django/steering-council I would love to have you add your thoughts to this PR as this would be the first time we mention DEPs in the contributing docs

Reply:
@sarahboyce Thank you reviewing.
I have updated changes, and this is what it looks like now 

<img width="1344" alt="Screenshot 2024-09-12 at 2 32 43 PM" src="https://github.com/user-attachments/assets/3cd511e5-5e7c-411d-ab1e-d310f9ce3244">

I’m wondering if the list items are too wordy because we have some paragraphs within them, like in DEP, where we have about three. Is this okay, or should we consider making it more concise? I’m also thinking it might be better to make the list items more descriptive. What do you think?